examples/user_guide/12-Responding_to_Events.ipynb

```diff
@@ -344,26 +344,73 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from holoviews.streams import Params\n",
+    "class BankAccount(param.Parameterized):\n",
+    "    balance = param.Number(default=0, doc=\"Bank balance in USD\")\n",
     "\n",
-    "class Style(param.Parameterized):\n",
-    "\n",
-    "    cmap = param.ObjectSelector(default='viridis', objects=['viridis', 'plasma', 'magma'])\n",
-    "\n",
-    "    color_levels = param.Integer(default=255, bounds=(1, 255))\n",
-    "\n",
-    "style = Style()\n",
+    "janes_account = BankAccount(balance=300) # An instance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can link parameter changes straight to DynamicMap callable parameters using the streams dictionary format (HoloViews version >= 1.14.2):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "jane_dmap = hv.DynamicMap(lambda dollars: hv.Text(0.5,0.5,\"Jane's balance: $%s\" % dollars), \n",
+    "              streams=dict(dollars=janes_account.param.balance))\n",
+    "jane_dmap.opts(height=100, xaxis=None, yaxis=None)"
```

Comment:
Still finding examples using `Text` on a plot a bit contrived, a hv.Table would definitely be preferable, since it's something you might actually do.

Reply:
Sure, I'll update to a `Table` example shortly.

Reply:
Updated in c2062e9f5

Reply:
Resolving this convo though happy to take any other improvements you have for this example.